impeller/entity/contents/content_context.cc

```diff
@@ -312,7 +310,13 @@ ContentContext::ContentContext(
       CreateDefaultPipeline<YUVToRGBFilterPipeline>(*context_);
   porter_duff_blend_pipelines_[default_options_] =
       CreateDefaultPipeline<PorterDuffBlendPipeline>(*context_);
-
+  // GLES Android only shader.
+#ifdef FML_OS_ANDROID
```

Comment:
Nit: Gate on building the GLES backend, not Android? This will probably come in handy for other desktop/embedded GL contexts. (Even in the case that we don't ship desktop GL for Flutter)

Reply:
Done